nursery/register-url.yml

```diff
@@ -0,0 +1,11 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: register URL
```

Comment:
can we add a bit more context here? like "register URL with web server" maybe?

Reply:
good catch - how about `register URL with HTTP server`?

Reply:
sounds great